extra_foam/gui/ctrl_widgets/image_transform_ctrl_widget.py

```diff
@@ -176,11 +176,6 @@ class ImageTransformCtrlWidget(_AbstractCtrlWidget):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
-        self._ma_window_le = SmartLineEdit("1")
```

Comment:
why the moving average was removed from image_transform?

Reply:
The moving average was put in image_transform because of the lack of a global one. Now it is redundant and can be safely removed. 

By the way, this moving average only applies in image_transform and will not affect other analysis. Therefore, it is a bit misleading.

Reply:
Alright. Thanks